<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayDialog</class>
 <widget class="QDialog" name="PayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>355</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>518</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>支付</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/icons/wallet.png</normaloff>:/new/icons/wallet.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(233, 255, 224);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(130, 205, 255);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

</string>
     </property>
     <property name="text">
      <string>选择一张银行卡支付</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget{
	border:0.5px solid white;
	border-radius:6px;
	
	background-color: rgb(233, 255, 224);
	min-width:80px;
	font-family:&quot;Microsoft YaHei&quot;;
	font-size:11pt;
	font-weight:bold;
	color:white;
}

</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="waitGifL">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(130, 205, 255);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

</string>
       </property>
       <property name="text">
        <string>已享受折扣</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="waitL">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(130, 205, 255);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

</string>
       </property>
       <property name="text">
        <string>七天无理由退货</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="moneyL">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(130, 205, 255);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

</string>
       </property>
       <property name="text">
        <string>共需支付：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="payB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>支付</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelB">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 0.5px solid white;
	border-radius: 6px;
	background-color: rgb(90, 194, 198);
	min-width: 80px;
	font-family: &quot;Microsoft YaHei&quot;;
	font-size: 11pt;
	font-weight: bold;
	color: white;
}

QPushButton:hover {
	background-color: #1fab89;
}

QPushButton:pressed {
	background-color: #178f75;
}

QPushButton:disabled {
	background-color: gray;
	color: lightgray;
}
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
